common/utils_test.go

```diff
@@ -1,22 +1,22 @@
-package nozzle_test
+package common_test
```

Comment:
`package common` ?

Reply:
that's ok, that allows you to test just exported identifiers (Black-box Testing).